src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -171,6 +171,7 @@
     <PlatformManifestFileEntry Include="libmonosgen-2.0.dylib" IsNative="true" />
     <PlatformManifestFileEntry Include="llc" IsNative="true" />
     <PlatformManifestFileEntry Include="mono-aot-cross" IsNative="true" />
+    <PlatformManifestFileEntry Include="mono-aot-cross.exe" IsNative="true" FallbackFileVersion="6.0.0" />
```

Comment:
we should follow-up to see why the file version doesn't get set in the .exe. Hardcoding the version here will just bite us once the repo moves to 7.0.0

Reply:
The file version is set by this change here https://github.com/dotnet/runtime/pull/49830/commits/4ca80bbd69c4ac30cc7a9a2062e898d941d75048. IIRC it is mandatory to have the fallback version. As we are not going to ship windows build, it hopefully doesn't matter which version we set as the fallback version.

I will try to build it without the fallback version set to see whether it will still fail the build. It is possible that I was only getting failures before adding the version to the `.exe` file.

Reply:
@radekdoulik did you have a chance to try a build without the `FallbackFileVersion`?

Reply:
Yes, looks like it is not needed anymore. https://github.com/dotnet/runtime/pull/51229

Let see if it still builds with the restored platform matrix.